README.md

```diff
@@ -4,4 +4,36 @@ This is the code behind https://crowdstrike.github.io/community/
 On that webpage, you can find content such as:
 - [List of CrowdStrike's Open Source Projects](https://crowdstrike.github.io/community/)
 - [Guidance for CrowdStrike Engineering](https://crowdstrike.github.io/community/open-source-policy/#engineering)
-- [Guidance for All Other Departments/Orgs](https://crowdstrike.github.io/community/open-source-policy/#everyone-else)
\ No newline at end of file
+- [Guidance for All Other Departments/Orgs](https://crowdstrike.github.io/community/open-source-policy/#everyone-else)
+
+
+
+## Developing the Site
```

Comment:
```suggestion
## Building

To build from the repo after cloning:
```

Reply:
`build` eludes to a one time thing, like a compilation. Prefer "Developing" since it's more of a continuous process.